src/utils/types.js

```diff
@@ -53,3 +53,11 @@ export const wholeNumber = function(props, propName, componentName) {
     );
   }
 };
+
+export const analyticsConfig = PropTypes.shape({
+  searchStateHeader: bool,
```

Comment:
@ShahAnuj2610 You don't implement this prop, You need the set the search configuration in the analytics library.

Reply:
@ShahAnuj2610 How is this resolved?